<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="14" hostname="DESKTOP-AK61BLR" failures="5" tests="5" name="Default test(failed)" time="3.736" errors="0" timestamp="2021-10-17T20:04:04 MSK">
  <testcase classname="com.arekhava.languageschool.model.service.impl.CatalogServiceImplTest" name="changeCourseDataNegativeTest" time="0.084">
    <failure type="com.arekhava.languageschool.model.service.InvalidDataException" message="invalid data">
      <![CDATA[com.arekhava.languageschool.model.service.InvalidDataException: invalid data
at com.arekhava.languageschool.model.service.impl.CatalogServiceImpl.changeCourseData(CatalogServiceImpl.java:71)
at com.arekhava.languageschool.model.service.impl.CatalogServiceImplTest.changeCourseDataNegativeTest(CatalogServiceImplTest.java:107)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- changeCourseDataNegativeTest -->
  <testcase classname="com.arekhava.languageschool.model.service.impl.CatalogServiceImplTest" name="changeCourseDataPositiveTest" time="0.005">
    <failure type="com.arekhava.languageschool.model.service.InvalidDataException" message="invalid data">
      <![CDATA[com.arekhava.languageschool.model.service.InvalidDataException: invalid data
at com.arekhava.languageschool.model.service.impl.CatalogServiceImpl.changeCourseData(CatalogServiceImpl.java:71)
at com.arekhava.languageschool.model.service.impl.CatalogServiceImplTest.changeCourseDataPositiveTest(CatalogServiceImplTest.java:101)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- changeCourseDataPositiveTest -->
  <testcase classname="com.arekhava.languageschool.model.service.impl.CatalogServiceImplTest" name="changeCourseDataServiceExceptionTest" time="0.008">
    <failure type="org.testng.TestException" message="
Expected exception of type class com.arekhava.languageschool.model.service.ServiceException but got com.arekhava.languageschool.model.service.InvalidDataException: invalid data">
      <![CDATA[org.testng.TestException: 
Expected exception of type class com.arekhava.languageschool.model.service.ServiceException but got com.arekhava.languageschool.model.service.InvalidDataException: invalid data
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
Caused by: com.arekhava.languageschool.model.service.InvalidDataException: invalid data
at com.arekhava.languageschool.model.service.impl.CatalogServiceImpl.changeCourseData(CatalogServiceImpl.java:71)
at com.arekhava.languageschool.model.service.impl.CatalogServiceImplTest.changeCourseDataServiceExceptionTest(CatalogServiceImplTest.java:113)
... 22 more
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- changeCourseDataServiceExceptionTest -->
  <testcase classname="com.arekhava.languageschool.model.service.impl.CatalogServiceImplTest" name="addProductPositiveTest" time="0.011">
    <failure type="com.arekhava.languageschool.model.service.InvalidDataException" message="invalid data">
      <![CDATA[com.arekhava.languageschool.model.service.InvalidDataException: invalid data
at com.arekhava.languageschool.model.service.impl.CatalogServiceImpl.addCourse(CatalogServiceImpl.java:50)
at com.arekhava.languageschool.model.service.impl.CatalogServiceImplTest.addProductPositiveTest(CatalogServiceImplTest.java:82)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- addProductPositiveTest -->
  <testcase classname="com.arekhava.languageschool.model.service.impl.CatalogServiceImplTest" name="addProductServiceExceptionTest" time="0.01">
    <failure type="org.testng.TestException" message="
Expected exception of type class com.arekhava.languageschool.model.service.ServiceException but got com.arekhava.languageschool.model.service.InvalidDataException: invalid data">
      <![CDATA[org.testng.TestException: 
Expected exception of type class com.arekhava.languageschool.model.service.ServiceException but got com.arekhava.languageschool.model.service.InvalidDataException: invalid data
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
Caused by: com.arekhava.languageschool.model.service.InvalidDataException: invalid data
at com.arekhava.languageschool.model.service.impl.CatalogServiceImpl.addCourse(CatalogServiceImpl.java:50)
at com.arekhava.languageschool.model.service.impl.CatalogServiceImplTest.addProductServiceExceptionTest(CatalogServiceImplTest.java:94)
... 22 more
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- addProductServiceExceptionTest -->
  <testcase classname="com.arekhava.languageschool.model.service.impl.CatalogServiceImplTest" name="takeCoursesFromCategoryServiceExceptionTest" time="0.0">
    <ignored/>
  </testcase> <!-- takeCoursesFromCategoryServiceExceptionTest -->
  <testcase classname="com.arekhava.languageschool.model.service.impl.CatalogServiceImplTest" name="takeCoursesWithNameServiceExceptionTest" time="0.0">
    <ignored/>
  </testcase> <!-- takeCoursesWithNameServiceExceptionTest -->
  <testcase classname="com.arekhava.languageschool.model.service.impl.CatalogServiceImplTest" name="takeAllLanguagesNegativeTest" time="0.0">
    <ignored/>
  </testcase> <!-- takeAllLanguagesNegativeTest -->
  <testcase classname="com.arekhava.languageschool.model.service.impl.CatalogServiceImplTest" name="changeCourseDataInvalidDataExceptionTest" time="0.0">
    <ignored/>
  </testcase> <!-- changeCourseDataInvalidDataExceptionTest -->
  <testcase classname="com.arekhava.languageschool.model.service.impl.CatalogServiceImplTest" name="takeCoursesWithNamePositiveTest" time="0.0">
    <ignored/>
  </testcase> <!-- takeCoursesWithNamePositiveTest -->
  <testcase classname="com.arekhava.languageschool.model.service.impl.CatalogServiceImplTest" name="addProductInvalidDataExceptionTest" time="0.0">
    <ignored/>
  </testcase> <!-- addProductInvalidDataExceptionTest -->
  <testcase classname="com.arekhava.languageschool.model.service.impl.CatalogServiceImplTest" name="takeCoursesFromCategoryNegativeTest" time="0.0">
    <ignored/>
  </testcase> <!-- takeCoursesFromCategoryNegativeTest -->
  <testcase classname="com.arekhava.languageschool.model.service.impl.CatalogServiceImplTest" name="takeCoursesWithNameNegativeTest" time="0.0">
    <ignored/>
  </testcase> <!-- takeCoursesWithNameNegativeTest -->
  <testcase classname="com.arekhava.languageschool.model.service.impl.CatalogServiceImplTest" name="takeAllLanguagesServiceExceptionTest" time="0.0">
    <ignored/>
  </testcase> <!-- takeAllLanguagesServiceExceptionTest -->
  <testcase classname="com.arekhava.languageschool.model.service.impl.CatalogServiceImplTest" name="takeCourseByIdPositiveTest" time="0.0">
    <ignored/>
  </testcase> <!-- takeCourseByIdPositiveTest -->
  <testcase classname="com.arekhava.languageschool.model.service.impl.CatalogServiceImplTest" name="takeAllLanguagesPositiveTest" time="0.0">
    <ignored/>
  </testcase> <!-- takeAllLanguagesPositiveTest -->
  <testcase classname="com.arekhava.languageschool.model.service.impl.CatalogServiceImplTest" name="takeCourseByIdNegativeTest" time="0.0">
    <ignored/>
  </testcase> <!-- takeCourseByIdNegativeTest -->
  <testcase classname="com.arekhava.languageschool.model.service.impl.CatalogServiceImplTest" name="takeCoursesLanguagePositiveTest" time="0.0">
    <ignored/>
  </testcase> <!-- takeCoursesLanguagePositiveTest -->
  <testcase classname="com.arekhava.languageschool.model.service.impl.CatalogServiceImplTest" name="takeCourseByIdServiceExceptionTest" time="0.0">
    <ignored/>
  </testcase> <!-- takeCourseByIdServiceExceptionTest -->
</testsuite> <!-- Default test(failed) -->
